libmime/patch.go

```diff
@@ -14,13 +14,20 @@ func dontOverride(ext string, mimeType string) (newExt string, newMimeType strin
 }
 
 // Patch patches the mime types Go uses by calling mime.AddExtensionType on
-// each from a private list in this package. Optionally provide a non-nil
-// Overrider to override any mime type defined in the list.
-func Patch(override Overrider) {
+// each from a private list in this package.  Both parameters are optional.
+// Provide a non-nil Overrider to override any mime type defined in the list.
+// Provide a non-nil additional map (ext->mimeType) to add additional mime
+// types.
```

Comment:
Maybe mention that the `overrider` can also override items in the `additional` list?  Might make sense to swap the order of the params too, since that's the case.

Reply:
Hmm that's actually not the case now but I think it makes sense. So will make it so.

Reply:
Oh sorry that's actually the case. Will update comment/parameter order. Thanks!